release_notes/374-release-notes.adoc

```diff
@@ -0,0 +1,345 @@
+:_content-type: ASSEMBLY
+[id="release-notes-374"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.74
+include::modules/common-attributes.adoc[]
+:context: release-notes-374
+
+toc::[]
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime stages of the application lifecycle. It deploys in your infrastructure and integrates with your DevOps tools and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+.Release dates
+[options="header"]
+|====
+|{product-title-short} version |Released on
+|`3.74` |27 February 2023
+|====
+
+[id="about-this-release-374"]
+== About this release
+
+{product-title-short} 3.74 includes the following new features, improvements, and updates:
+
+* <<ibm-power-z-374>>
+* <<clair-v4-374>>
+* <<network-graph-2-374>>
+* <<global-search-updates-374>>
+* <<syslog-integration-374>>
+* <<troubleshooting-collector-374>>
+* <<scale-performance-improvements-374>>
+* <<postgresql-technology-preview-374>>
+** <<collections-374>>
+** <<policy-categories-374>>
+* <<bug-fixes-374>>
+
+[id="new-features-374"]
+== New features
+
+[id="ibm-power-z-374"]
+=== {ibm-power} and {ibm-zsystems} support for secured clusters
+
+{product-title-short} version 3.74 extends support for {product-title-short} secured clusters for {osp} 4.10, 4.11, and 4.12 to {ibm-zsystems} and {ibm-power} architectures. With {product-title-short} version 3.74, you can install secured cluster services on {osp} on {ibm-zsystems} and {ibm-power} by using the {product-title-short} Operator. 
```

Comment:
```suggestion
{product-title-short} version 3.74 extends support for {product-title-short} secured clusters for {osp} 4.10, 4.11, and 4.12 to {ibm-power}(ppc64le), {ibm-zsystems}(s390x), and {ibm-linuxone}(s390x) architectures. With {product-title-short} version 3.74, you can install secured cluster services on {osp} on {ibm-power}, {ibm-zsystems}, and {ibm-linuxone} by using the {product-title-short} Operator. 
```

Reply:
Silke, thank you for confirming via Slack - I've sent it back to our product team for one last review to make sure the change is OK with them.

Reply:
we are good to go!